http/src/main/resources/reference.conf

```diff
@@ -184,7 +184,7 @@ azure {
         type = "CustomScript",
         version = "2.1",
         minor-version-auto-upgrade = true,
-        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/74c55827dd7fcefe56bbff14de1aefb3622e849e/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
+        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/140d9c5fa6723fd92ce30d75595a6c2f9f5db290/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
```

Comment:
what do these hashes represent?

Reply:
it's just permlink for [this file](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/azure_vm_init_script.sh) so that it won't change...